app/assets/javascripts/components/profile/profile_cards.es6.jsx

```diff
@@ -14,8 +26,8 @@ class ProfileCards extends Component {
   render() {
     return (
       <div style = {this.styles.container}>
-        <ProfilePreview />
-        <ProfileOptions />
+        <ProfilePreview currentUser={this.props.currentUser} />
```

Comment:
You can use `{...this.props}` (which passes down all of this props to the child component) here!